omod/src/test/java/org/openmrs/module/idgen/web/resource/LogEntryResourceTest.java

```diff
@@ -0,0 +1,69 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.idgen.web.resource;
+
+import java.util.Date;
+import org.junit.Before;
+import org.omg.CORBA.Context;
```

Comment:
What are you using the above Context for?

Reply:
That was a wrong import. Intention was to import 
>org.openmrs.api.context.Context;
Fixed it and it sorts the issue with getUserService() & getService()